test/unit/test.realm.js

```diff
@@ -101,21 +101,10 @@ describe('=> Realm', () => {
     // clean all prev data in users
     await realm.query('TRUNCATE TABLE users');
     assert.rejects(async () => {
-      await realm.transaction(function *() {
-        yield realm.query(`INSERT INTO
-        users (gmt_create, email, nickname)
-        VALUES (?, ?, ?)`, [
-          new Date(),
-          email,
-          'Thor',
-        ]);
-        yield realm.query(`INSERT INTO
-        users (gmt_create, email, nickname)
-        VALUES (?, ?, ?)`,[
-          new Date(),
-          email,
-          'Loki',
-        ]);
+      await realm.transaction(function *({ connection }) {
+        const sql = 'INSERT INTO users (gmt_create, email, nickname) VALUES (?, ?, ?)';
+        yield realm.query(sql, [ new Date(), email, 'Thor' ], { connection });
+        yield realm.query(sql, [ new Date(), email, 'Loki' ], { connection });
```

Comment:
currently, automatic connection injection in generator function is only possible if Spell were involved. Hence if it's raw query, we need to pass on `{ connection }` manually.